packages/grid/_modules_/grid/hooks/features/filter/useGridFilter.ts

```diff
@@ -205,23 +191,18 @@ export const useGridFilter = (
   const deleteFilter = React.useCallback(
     (item: GridFilterItem) => {
       logger.debug(`Deleting filter on column ${item.columnField} with value ${item.value}`);
-      let hasNoItem = false;
       setGridState((state) => {
         const items = [...state.filter.items.filter((filterItem) => filterItem.id !== item.id)];
-        hasNoItem = items.length === 0;
         const newState = {
           ...state,
           filter: { ...state.filter, items },
         };
         return newState;
       });
-      if (hasNoItem) {
-        upsertFilter({});
```

Comment:
fixed

Reply:
Thinking about letting delete the last item like airtable.
What do you think? cc @oliviertassinari

Reply:
snooze :)

Reply:
> Thinking about letting delete the last item like airtable.
What do you think? cc @oliviertassinari

Yeah, big 💯 for this, it would already be a good step forward to improve the UX of the filter.